src/ResourceManager/ApplicationInsights/Commands.ApplicationInsights/Get-AzureRmApplicationInsights.md

```diff
@@ -0,0 +1,132 @@
+---
+external help file: Microsoft.Azure.Commands.ApplicationInsights.dll-Help.xml
+Module Name: AzureRM.ApplicationInsights
+online version: 
+schema: 2.0.0
+---
+
+# Get-AzureRmApplicationInsights
+
+## SYNOPSIS
+Get application insights resources
+
+## SYNTAX
+
+### ResourceGroupParameterSet
+```
+Get-AzureRmApplicationInsights [[-ResourceGroupName] <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ComponentNameParameterSet
+```
+Get-AzureRmApplicationInsights [-ResourceGroupName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Get-AzureRmApplicationInsights [-ResourceId] <ResourceIdentifier> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get application insights resources in a resource group or specific resource
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Get-AzureRmApplicationInsights -ResourceGroupName "testgroup" -Name "test"
+```
+
+Get application insights resoruce named "test" in resoruce group "testgroup"
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure.
+
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Name
+Application Insights Resource Name.
+
+```yaml
+Type: String
+Parameter Sets: ComponentNameParameterSet
+Aliases: ApplicationInsightsComponentName, ComponentName
+
+Required: True
+Position: 1
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Resource Group Name.
+
+```yaml
+Type: String
+Parameter Sets: ResourceGroupParameterSet
+Aliases: 
+
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+```yaml
+Type: String
+Parameter Sets: ComponentNameParameterSet
+Aliases: 
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceId
+Application Insights Component Resource Id.```yaml
```

Comment:
@frankguodongchen this is a bug in `platyPS`, you will need to go through your markdown files and ensure that there is no ` ```yaml ` string at the end of the parameter description. These strings need to go on a line by themselves (see above parameters as an example).